tests/destructor/t9440.nim

```diff
@@ -0,0 +1,51 @@
+discard """
+  matrix: "--gc:refc; --gc:orc; --gc:arc"
+  output: '''
+()
+Destroyed
+()
+Destroyed
+()
+Destroyed
+end
+-------------------------
+()
+Destroyed
+end
+'''
+
+"""
+# bug 9440
```

Comment:
```suggestion
# bug #9440
```

(as documented in contributing.html; the rationale is it makes it more searchable, and also it makes the link clickable in github and other tools)